src/containers/FactoryLoader.tsx

```diff
@@ -24,7 +24,9 @@ import { WorkspaceStatus } from '../services/helpers/types';
 import { sanitizeLocation } from '../services/helpers/location';
 import { merge } from 'lodash';
 
-const WS_ATTRIBUTES_TO_SAVE: string[] = ['workspaceDeploymentLabels', 'workspaceDeploymentAnnotations'];
+const WS_ATTRIBUTES_TO_SAVE: string[] = ['workspaceDeploymentLabels', 'workspaceDeploymentAnnotations', 'policies.create'];
+
+type CreatePolicy = 'perclick' | 'peruser' | 'peraccount';
```

Comment:
I have a difficulty with understanding peruser and peraccount...

Per account - means per user or per organization depending on the workspace owner.

1. When I do 
`http://localhost:3000/#/load-factory?url=https://github.com/eclipse/che-dashboard.git&policies.create=peruser` it creates a workspace and reuse it after I open it over and over.

2. After I do `http://localhost:3000/#/load-factory?url=https://github.com/eclipse/che-dashboard.git&policies.create=peraccount`, My workspace is still created under my personal account, the new one, the one from step 1 is not reused.

As far as I see that's not documented and since we deprecated organizations for time being, maybe it would be better to go with perclick, peruser at this point.

Maybe I'm just missing something. I'll read the code tomorrow to understand what peraccount actually does.

Reply:
I have removed 'peraccount' at all.